src/components/BalancePage/BalancePage.jsx

```diff
@@ -4,22 +4,42 @@ import { HOME } from '../../routes';
 
 import NavLink from '../NavLinks/NavLink.jsx';
 
-const BalancePage = () => (
-  <div className="balance__page">
-    <h1 className="balance__page_balance">
-      Your current balance is:
-      <span className="main__page_header-span"> 0,00 💲</span>
-    </h1>
-    <h3 className="balance__page_spent">
-      Total spent:
-      <span className="main__page_header-span"> 1034074028,00 💲</span>
-    </h3>
-    <h3 className="balance__page_received">
-      Total received:
-      <span className="main__page_header-span"> 1034074028,00 💲</span>
-    </h3>
-    <NavLink text={`Home`} route={HOME} />
-  </div>
-);
+const { useSelector } = require('react-redux');
+
+const BalancePage = () => {
+  const list = useSelector(state => state.transactionsList.transactionsList);
```

Comment:
I can advice move `state => state.transactionsList.transactionsList` to  `https://github.com/vitcool/testnet/blob/master/src/redux/modules/transactionsList/selectors.js` as
```
const getTransactionsList = (state) => state.transactionsList.transactionsList;
```
and use it here
```
const list = useSelector(getTransactionsList);
```

Reply:
Fixed